### DISCUSSION TITLE
image only

### DISCUSSION THREAD

**Discussion by TomaszMyszak** [abraunegg/onedrive#1450] [Q&A] [answered]
I installed the program and started the synchronization - it works. It just downloads my entire files to disk. How to configure image only download (so that it takes up about 10% of disk space - just like in windows)

**Comment by abraunegg** [comment, marked as answer]
@TomaszMyszak 
You are looking for what is known as 'on-demand' files. Please refer to https://github.com/abraunegg/onedrive/issues/757
